backend/app/common/openai_responses.py

```
@@ -0,0 +1,330 @@
+    instructions = body.get("instructions")
+    input_value = body.get("input")
+
+    # Some clients may still send `messages`; treat as-is.
+    messages = body.get("messages")
+    if isinstance(messages, list):
+        chat_messages = messages
+    else:
+        chat_messages = _coerce_input_to_messages(input_value)
+
+    if isinstance(instructions, str) and instructions:
+        chat_messages = [{"role": "system", "content": instructions}] + chat_messages
```

Ruff (0.14.13)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)